### PR TITLE
Add sonatype related settings

### PR DESCRIPTION
**Pull request by mdedetrich**
Sets the necessary sonatype settings for publishing to work

### CODE REVIEW

**Review by ennru — APPROVED** [review]
Thank you. Was just looking if we had those...

File: build.sbt [hunk 1]
```diff
@@ -58,6 +58,9 @@ ThisBuild / githubWorkflowPublish := Seq(
   )
 )
 
+sonatypeProfileName := "com.lightbend.paradox"
```
Comment (raboof) [inline build.sbt]: I think this should be `com.lightbend` - or do we want to introduce `com.lightbend.paradox` to Sonatype?

```suggestion
sonatypeProfileName := "com.lightbend"
```

**Review by raboof — APPROVED** [review]
:+1: , one comment

**Comment by mdedetrich** [comment]
@raboof I copied it from `com.lightbend.paradox` from https://github.com/lightbend/sbt-paradox-apidoc

**Comment by raboof** [comment]
> @raboof I copied it from `com.lightbend.paradox` from https://github.com/lightbend/sbt-paradox-apidoc

That indeed has `com.lightbend.paradox` as the `organization`, but I don't think it is configured to publish to sonatype at all yet? I don't see any artifacts at https://repo1.maven.org/maven2/com/lightbend/paradox/ at least...

**Comment by mdedetrich** [comment]
Yeah just realized, you can ignore my earlier comment.

**Comment by ennru** [comment]
Now Sonatype worries about non-present JavaDoc https://github.com/lightbend/sbt-paradox-project-info/runs/5511404346?check_suite_focus=true#step:9:90

**Pull request merged**